pydantic/typing.py

```diff
@@ -193,3 +193,12 @@ def update_field_forward_refs(field: 'Field', globalns: Any, localns: Any) -> No
     if field.sub_fields:
         for sub_f in field.sub_fields:
             update_field_forward_refs(sub_f, globalns=globalns, localns=localns)
+
+
+def get_class(type_: AnyType) -> Optional[Union[AnyType, bool]]:
+    origin = getattr(type_, '__origin__', None)
+    if origin is not None and issubclass(origin, Type):  # type: ignore
+        if type_.__args__ is None or not isinstance(type_.__args__[0], type):
+            return True
+        return type_.__args__[0]
+    return None
```

Comment:
no need for `return None`

Reply:
mypy complains without a return statement

Reply:
okay